### DISCUSSION TITLE
How to access management apis from other backend service?

### DISCUSSION THREAD

**Discussion by lovepocky** [logto-io/logto#2235] [Q&A]
as title

**Comment by gao-sun** [comment]
Edit: please check [Interact with Management API](https://docs.logto.io/docs/recipes/interact-with-management-api/).

good question. we're writing docs about this

**Reply by gao-sun** [reply]
will paste the link here once it's finished

**Reply by gao-sun** [reply]
@lovepocky please see the updated answer with the link
